src/routes/blog/posts/introducing-files-v2-1.svx

```diff
@@ -0,0 +1,61 @@
+---
+title: Introducing Files v2.1
+description: Today, we're pleased to announce that Files v2.1 is now available. We focused on improving the performance and behavior of existing features in this update to make it even easier for you to use the app.
+thumbnail: /blog-resources/filesv2/FilesV2Banner.jpg
+date: 2022-1-17
+author: yaichenbaum
+---
+
+Today, we're pleased to announce that [Files v2.1]({'https://click.linksynergy.com/deeplink?id=pVFXwNFwveg&mid=24542&murl=https%3A%2F%2Fwww.microsoft.com%2Fstore%2Fapps%2F9NGHP3DX8HDX'}) is now available. We focused on improving the performance and behavior of existing features in this update to make it even easier for you to use the app.
+
+## Settings Files as the default file manager
+
+We added an experimental option in v2.0.33 to set Files as the default file manager. This option is still experimental but we improved the reliability based on your feedback and we recommend giving it a try.
```

Comment:
nit: Maybe we could rephrase this a bit to make it more clear that this was already present? First time I read this, I was confused because the blog is about 2.1 and not 2.0.33. But maybe this is just me here.

Reply:
The new formulation is great, feel free to merge this @yaichenbaum if you are ready.

Reply:
I still have a couple screenshots to upload and to finalize the release.